Notify/WEB/ClientApp/src/components/NotificationListPage/NotificationListPage.js

```diff
@@ -1,67 +1,60 @@
-﻿import React, { Component } from 'react';
+﻿import React, { Component } from 'react'
+import { connect } from 'react-redux'
 
 
-import SearchPanel from '../SearchPanel/SearchPanel';
-import NotificationList from '../NotificationList/NotificationList';
-import Notification from '../Notification/Notification';
+import SearchPanel from './SearchPanel/SearchPanel'
+import NotificationList from './NotificationList/NotificationList'
+import Notification from './Notification/Notification'
+import { getNotificationList } from '../../actions/notificationList'
 
-export default class NotificationListPage extends Component {
+
+class NotificationListPage extends Component {
 
     state = {
-        sort: {
-            sorting: 0,
-            filterby: null,
-            searchText: null,
-            page: 1,
-            pageSize: 10
-        },
-        notificationId : null
+        userId: localStorage.getItem("userId"),
+        sorting: 0,
+        filterBy: null,
+        page: 1,
+        pageSize: 10,
+        searchText: null
+    }
```

Comment:
sorting is just about order of some data, don't mix all in one: paging, sorting, filtering